callmetadata.schema.json

```diff
@@ -5,6 +5,27 @@
     "description": "Meta data supporting dialtone, ringdown, and shoutdown calls",
     "type": "object",
     "definitions": {
+        "state": {
```

Comment:
What if we call it event which will support all kind of events(even those that are system specific)? Also the listed examples in description are events(always something that happened) and not states.

Reply:
Shall we call the top-level Event and then have a state within it?